src/System.Device.Gpio/System/Device/Gpio/Drivers/RaspberryPi3Driver.Linux.cs

```diff
@@ -321,6 +321,9 @@ private void SetInputPullModePi4(int pinNumber, PinMode mode)
             bits &= ~(3u << shift);
             bits |= (pull << shift);
             gpioReg->GPPUPPDN[(pinNumber >> 4)] = bits;
+            for (int i = 0; i < 150; i++)
```

Comment:
you will need no optimizations to get that to work, otherwise this might get optimized away

Reply:
As you say it... I guess that's true. I will have to retest in release.

Reply:
even if it works on release you can't guarantee this will not just stop working in the future or work differently on different runtime